js/arbeidsrettet-oppfolging/MerVeiledning.js

```diff
@@ -48,27 +50,43 @@ class TrengerMerVeiledningRad extends Component {
         const { bekrefter, bekreftingFeilet } = this.props;
         return (
             /* eslint-disable jsx-a11y/no-static-element-interactions */
-            <div className="infoside-fo__rad infoside-fo__rad--graa">
-                <div className="begrensning">
-                    <Systemtittel className="blokk-s">{getLedetekst('infoside-fo.veiledning.overskrift')}</Systemtittel>
-                    <div className="redaksjonelt-innhold blokk">{getLedetekst('infoside-fo.veiledning.tekst')}</div>
-                    <Feilstripe vis={bekreftingFeilet} className="blokk-s" />
-                    <div className="infoside-fo__knapperad">
-                        <Knapp onClick={this.handleNeiBtnClicked} disabled={bekrefter}>
-                            {getLedetekst('infoside-fo.knapp-nei')}
-                        </Knapp>
-                        <Hovedknapp onClick={this.handleJaBtnClicked} spinner={bekrefter} disabled={bekrefter}>
-                            {getLedetekst('infoside-fo.knapp-ja')}
-                        </Hovedknapp>
-                    </div>
+            <div className="panel panel--merVeiledning blokk">
+                <div className="illustrertTittel">
+                    <img className="illustrertTittel__img" src={hentArbeidsrettetOppfolgingBilde('veien-videre.svg')} alt="Veien videre" />
+                    <h2 className="illustrertTittel__tittel illustrertTittel__tittel--innholdstittel">
+                        {getLedetekst('ao.mer-veiledning.tittel')}
+                    </h2>
+                </div>
+                <div
+                    className="redaksjonelt-innhold blokk"
+                    dangerouslySetInnerHTML={getHtmlLedetekst('ao.mer-veiledning.tekst')} />
+                <Feilstripe vis={bekreftingFeilet} className="blokk" />
+                <div className="merVeiledning__knapper">
+                    <Undertittel tag="h3" className="blokk">{getLedetekst('ao.mer-veiledning.sporsmal')}</Undertittel>
+                    {
+                        !bekrefter
+                            ? (<React.Fragment>
+                                <p className="blokk--xxs">
```

Comment:
En Knapp inni en `<p>`? Kan ikke knappen få stå for seg selv?

Reply:
nei, da den skal stå på en egen linje

Reply:
Kan man ikke gjøre det med styling? Blir det ikke litt rart å bruke semantikk for å gjøre stylingjobb?